recipes/bioconda-utils/meta.yaml

```diff
@@ -18,10 +18,10 @@ build:
 
 requirements:
   host:
-    - python >=3.6
+    - python >=3.7
```

Comment:
You can ignore this requirement in practice, I think it's a hold-over from some earlier testing.

Reply:
Possibly, but it doesn't hurt to go for 3.7.
For the test we definitely need to have the requirements in `meta.yaml` and `bioconda_utils-requirements.txt` in agreement, though.